cmd/helm-sync/main.go

```diff
@@ -108,6 +120,9 @@ func main() {
 
 	initialSync := true
 	failCount := 0
+	step := time.Duration(0)
```

Comment:
nit: no need to declare this variable up here, can be declared in the loop

Reply:
Got it, thanks